test/next/src/app/vercel/blob/form-body-upload.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import type { BlobResult } from '@vercel/blob';
+import { useRef, useState } from 'react';
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
@vvo I had to fix all the eslint issues in the nextjs example, including file names

Reply:
👍